.github/workflows/clear-runs.yml

```diff
@@ -0,0 +1,83 @@
+name: Clear outdated runs
+
+on: workflow_call
```

Comment:
I think you need to add the environment handling back in here

Reply:
Oops, the file was missed when committing